### PR TITLE
doc: update on cellular UDP sample

### PR DESCRIPTION
**Pull request by divipillai**
update on cellular UDP sample

### CODE REVIEW

**Comment by peknis** [comment]
@gmarull, the mysterious image compliance failure is also here inspite of the adjusted .checkpatch.conf (https://github.com/nrfconnect/sdk-nrf/pull/12638/files). Please check what is going on.

File: samples/cellular/udp/README.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Cellular: UDP
    :local:
    :depth: 2
 
-The UDP sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.
+The Cellular: UDP sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.
```
Comment (peknis) [inline samples/cellular/udp/README.rst:10]:
```suggestion
This sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.
```

File: samples/cellular/udp/README.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ These parameters can be set through the sample configuration file :file:`prj.con
 You can configure the frequency with which the packets are transmitted and the size of the UDP payload through the Kconfig system.
 In addition to setting of the above options, you can also set the various LTE parameters that are related to current consumption for adding low power behavior to the device.
 
-The UDP sample can be used to characterize the current consumption of the nRF91 Series SiP.
+The Cellular: UDP sample can be used to characterize the current consumption of the nRF91 Series SiP.
```
Comment (peknis) [inline samples/cellular/udp/README.rst:35]:
```suggestion
You can use this sample to characterize the current consumption of the nRF91 Series SiP.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/cellular/udp/README.rst (hunk 1 above)
Comment (divipillai) [reply samples/cellular/udp/README.rst:10]: The reason to explicitly mention the sample name as cellular: UDP is, we have a similar sample in the networking folder which has Wi-Fi support. https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/net/udp/README.html.

So, I think we could keep this as is?

File: samples/cellular/udp/README.rst (hunk 2 above)
Comment (divipillai) [reply samples/cellular/udp/README.rst:35]: may be : _You can use  Cellular: UDP sample ...._ ?

File: samples/cellular/udp/README.rst (hunk 1 above)
Comment (peknis) [reply samples/cellular/udp/README.rst:10]: I'd say we can still safely say "This sample", as the reader is possibly on this page when reading.

Comment (divipillai) [reply samples/cellular/udp/README.rst:10]: Discussed offline, and decided to keep the sample name at the first instance.

**Review by peknis — APPROVED** [review]

**Pull request merged**